datadog_checks_downloader/tests/test_downloader.py

```diff
@@ -335,7 +339,7 @@ def _do_download(package, version=None, root_layout_type="core"):
     # -vvvv:  INFO
     # -vvvvv: DEBUG
     cmd = [
-        "python",
+        sys.executable,
```

Comment:
Note: this change was necessary so that it uses the right `python` in Windows, otherwise it would be missing the dependencies.

Reply:
afaik this is also what the python docs recommend, so +1 on the change!